opal/mca/pmix/pmix112/pmix/NEWS

```diff
@@ -23,6 +23,39 @@ current release as well as the "stable" bug fix release branch.
 Master (not on release branches yet)
 ------------------------------------
 
+1.1.4
```

Comment:
Because we only worry about updating the NEWS when we do the official release, and we haven't done that yet. For the embedded code, it is irrelevant.